src/components/account/OrderListItem.js

```diff
@@ -30,7 +29,7 @@ class OrderListItem extends Component {
             {`Ship to ${item.customer_firstname} ${item.customer_lastname}`}
           </Text>
           <Text style={text} >
-            {`Order Total: ${priceSignByCode(item.order_currency_code)} ${item.grand_total}`}
+            {`Order Total: ${currencySymbol} ${item.grand_total}`}
```

Comment:
I have changed the logic, instead of determining currency symbol by using the currency code given in item, I am directly using default currency symbol, that I get from the currency API